File: Store.Domain/Order.cs
using System;
using System.Collections.Generic;
using Store.Domain.Identity;

namespace Store.Domain
{
    public class Order
    {
        public int Id { get; set; }

        public DateTime Date { get; set; }

        public decimal TotalPurchaseAmount { get; set; }

        public int CostumerId { get; set; }

        public Costumer Costumer { get; set; }

        public string SellerUser { get; set; }

        public List<OrderProduct> OrderProducts { get; set; }
    }
}

File: Store-App/src/app/_models/Order.ts
import { OrderProduct } from './OrderProduct';
import { Costumer } from './Costumer';


export interface Order {

    id: number;

    date: Date;

    totalPurchaseAmount: number;

    costumerId: number;

    costumer: Costumer;

    sellerUser: string;

    orderProducts: OrderProduct[];
}
